install_local.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+
+"""
+Helper script for installing spyder and external-deps locally in develop mode.
+"""
```

Comment:
```suggestion
"""Helper script for installing spyder and external-deps locally in editable mode."""

```

Use better, more precise terms and follow PEP 257.